### PR TITLE
add installer module

### PR DESCRIPTION
**Pull request by tornado404**

### CODE REVIEW

File: installer/charts/paddleflow-server/Chart.yaml [hunk 1]
```diff
@@ -1,24 +1,11 @@
...
-# This is the version number of the application being deployed. This version number should be
-# incremented each time you make changes to the application. Versions are not expected to
-# follow Semantic Versioning. They should reflect the version the application is using.
-# It is recommended to use it with quotes.
-appVersion: "1.16.0"
+home: paddleflow
+version: 0.0.91-rc68gc-tmp
+appVersion: 0.0.91-rc68gc-tmp
+apiVersion: v1
+description: baidu paddleflow-server.
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/Chart.yaml]: Remove internal sensitive information `baidu`.

File: installer/charts/paddleflow-server/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# demo-product helm chart
+
+baidu demo-product helm chart
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/README.md]: Sensitive information.

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml [hunk 3]
```diff
@@ -1,107 +1,409 @@
...
+          isCleanJob: true
+          isSkipCleanFailedJob: false
+          succeededJobTTLSeconds: 600
+          failedJobTTLSeconds: 3600
+        schedulerName: {{  $.Values.out_depend_msg.PF_SERVER_SCHEDULER_NAME  }}
+        clusterSyncPeriod: 30
+        jobLoopPeriod: 5
+        scalarResourceArray:
+          - "nvidia.com/gpu"
+          - "baidu.com/v100_cgpu"
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: Remove sensitive information.

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml [hunk 4]
```diff
@@ -1,107 +1,409 @@
...
+
+      flavour:
+        - name: flavour1
+          cpu: 1
+          mem: 1Gi
+        - name: flavour2
+          cpu: 1
+          mem: 1G
+          scalarResources:
+            "baidu.com/v100_cgpu": "1"
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: Remove sensitive information.

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml [hunk 5]
```diff
@@ -1,107 +1,409 @@
...
+        - name: flavour3
+          cpu: 1
+          mem: 8Gi
+          scalarResources:
+            "baidu.com/v100_cgpu": "1"
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: Sensitive information.

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml [hunk 6]
```diff
@@ -1,107 +1,409 @@
...
+        - name: flavour4
+          cpu: 1
+          mem: 8Gi
+          scalarResources:
+            "baidu.com/v100_cgpu": "1"
```
Comment (D0m021ng) [inline installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: Sensitive information.

File: installer/charts/pfs-csi-plugin/Chart.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+name: pfs-csi-plugin
+home: paddleflow
+version: 0.0.40-rc6eh8
+appVersion: 0.0.40-rc6eh8
+apiVersion: v1
+description: baidu pfs-csi-plugin.
```
Comment (D0m021ng) [inline installer/charts/pfs-csi-plugin/Chart.yaml]: Remove sensitive information.

File: installer/charts/pfs-csi-plugin/README.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# demo-product helm chart
+
+baidu demo-product helm chart
```
Comment (D0m021ng) [inline installer/charts/pfs-csi-plugin/README.md]: Sensitive information.

File: installer/charts/pfs-csi-plugin/values.yaml.template [hunk 9]
```diff
@@ -0,0 +1,297 @@
...
+  serviceName: ''
+  sidecar_container_msg:
+  - args: []
+    command:
+    - /sbin/tini
+    - --
+    - /bin/sh
+    - -c
+    - cd /home/paddleflow && /home/paddleflow/csi-plugin --unix-endpoint=$(CSI_ENDPOINT)
+      --node-id=$(KUBE_NODE_NAME) --log-dir=./log/csidriver --username=root --password=paddleflow
```
Comment (D0m021ng) [inline installer/charts/pfs-csi-plugin/values.yaml.template:207]: Sensitive Information `password`

File: installer/charts/pfs-csi-provisioner/README.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+# demo-product helm chart
+
+baidu demo-product helm chart
```
Comment (D0m021ng) [inline installer/charts/pfs-csi-provisioner/README.md]: Sensitive information.

File: installer/charts/paddleflow-server/README.md (hunk 2 above)
Comment (tornado404) [reply installer/charts/paddleflow-server/README.md]: done

File: installer/charts/paddleflow-server/Chart.yaml (hunk 1 above)
Comment (tornado404) [reply installer/charts/paddleflow-server/Chart.yaml]: done

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml (hunk 4 above)
Comment (tornado404) [reply installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: done

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml (hunk 6 above)
Comment (tornado404) [reply installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: done

File: installer/charts/pfs-csi-plugin/README.md (hunk 8 above)
Comment (tornado404) [reply installer/charts/pfs-csi-plugin/README.md]: done

File: installer/charts/pfs-csi-plugin/values.yaml.template (hunk 9 above)
Comment (tornado404) [reply installer/charts/pfs-csi-plugin/values.yaml.template:207]: done

File: installer/charts/pfs-csi-provisioner/README.md (hunk 10 above)
Comment (tornado404) [reply installer/charts/pfs-csi-provisioner/README.md]: done

File: installer/charts/pfs-csi-plugin/Chart.yaml (hunk 7 above)
Comment (tornado404) [reply installer/charts/pfs-csi-plugin/Chart.yaml]: done

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml (hunk 5 above)
Comment (tornado404) [reply installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: done

File: installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml (hunk 3 above)
Comment (tornado404) [reply installer/charts/paddleflow-server/templates/paddleflow-server-configmap.yaml]: done

**Pull request closed without merging**